### PR TITLE
WT-10765 Fix s_mentions script to work in ticket-named Git trees

### PR DESCRIPTION
**Pull request by agorrod**

### CODE REVIEW

File: dist/s_mentions [hunk 1]
```diff
@@ -3,6 +3,12 @@
 # Find comments related to the ticket potentially indicated by the branch name. The script never
 # fails, it only warns the caller there are comments that are potentially worth checking.
 
+# Testing notes:
+# Different users have different structures in their development trees. When making changes to this
+# script, please test a few different scenarios including. When there are multiple populated
```
Comment (luke-pearson) [inline dist/s_mentions:8]: I think this full stop should be a colon.

**Review by luke-pearson — APPROVED** [review]
LGTM.

File: dist/s_mentions (hunk 1 above)
Comment (agorrod) [reply dist/s_mentions]: Thanks - fixed.

**Comment by agorrod** [comment]
evergreen merge

**Pull request merged**